src/components/input-field/input-field.stories.js

```diff
@@ -84,6 +84,17 @@ export const Default = ({ block, error, success, small }) => html`
   />
 `;
 
+export const Disabled = ({ block, error, success, small }) => html`
+  <input
+    is="vl-input-field"
+    disabled
+    ?data-vl-block=${block}
+    ?data-vl-error=${error}
+    ?data-vl-success=${success}
+    ?data-vl-small=${small}
+    />
+`;
+
 // const Template = ({
```

Comment:
Hoort deze comment hier nog te staan?

Reply:
Goede vraag, uit te zoeken waarom dat gebeurd is en een comment bijschrijven. Heeft in ieder geval niets met deze test pr te maken (enkel prettier).